robottelo/ui/browser.py

```diff
@@ -42,12 +42,12 @@ def browser():
                 else webdriver.Chrome(
                     executable_path=settings.webdriver_binary)
             )
-        elif webdriver_name == 'ie':
+        elif webdriver_name == 'internet explorer':
             return (
                 webdriver.Ie() if settings.webdriver_binary is None
                 else webdriver.Ie(executable_path=settings.webdriver_binary)
             )
-        elif webdriver_name == 'edge':
+        elif webdriver_name == 'MicrosoftEdge':
```

Comment:
@rochacbruno maybe i don't get it, but your link shows opposite - webdriver's name is 'edge', browser's name is 'MicrosoftEdge', same here - https://github.com/SeleniumHQ/selenium/blob/master/py/selenium/webdriver/__init__.py#L23.
As far as i get it, you're proposing to use webdrivers naming according to browser names, not actual webdrivers names.